ekncontent/ekncontent/eknc-domain.c

```diff
@@ -739,36 +739,134 @@ static void
 query_state_free (gpointer data)
 {
   QueryState *state = data;
-  g_slist_free_full (state->models, g_object_unref);
+  g_clear_pointer (&state->models, g_ptr_array_unref);
   g_slice_free (QueryState, state);
 }
 
+typedef void (*ScatterTaskCountdownCompleteCallback)(gpointer user_data);
+typedef void (*ScatterTaskCountdownErrorCallback)(GError *error, gpointer user_data);
+
+typedef struct {
+  guint remaining;
+  ScatterTaskCountdownCompleteCallback callback;
+  ScatterTaskCountdownErrorCallback error_callback;
+  gpointer data;
+} ScatterTaskCountdown;
+
+static ScatterTaskCountdown *
+scatter_task_countdown_new (guint remaining,
+                            ScatterTaskCountdownCompleteCallback callback,
+                            ScatterTaskCountdownErrorCallback error_callback,
+                            gpointer data)
+{
+  ScatterTaskCountdown *countdown = g_new0(ScatterTaskCountdown, 1);
```

Comment:
Hmm, `g_new0` is a macro and my understanding is that macro calls can't contain any spaces between the macro name and arguments. Or maybe that was definitions - I'll just check with the space.